frontstage/common/eq_payload.py

```diff
@@ -87,6 +88,11 @@ def create_payload(self, case, ce, party_id: str, business_party_id: str, survey
         if employment_date := self._find_event_date_by_tag("employment", ce_events, ce_id, False):
             payload["survey_metadata"]["data"]["employment_date"] = employment_date
 
+        if ce["supplementaryDatasetEntity"]:
+            supplementary_dataset = json.loads(ce["supplementaryDatasetEntity"]["supplementaryDatasetJson"])
+            if form_type in supplementary_dataset["form_types"]:
```

Comment:
version control this please to not be enabled in prod

Reply:
Spoke with Mark and he believes we should be fine as we've essentially feature flagged the queue that we pick up the SD from.